### PR TITLE
Fix init phase of Miniprovisioning

### PR DESCRIPTION
**Pull request by selvakumaar5496**
# Problem Statement
- Init phase of utils Mini Provisioning failed

# Design
-  Fix init phase of utils Mini Provisioning 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [x] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [x] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -249,17 +224,17 @@ def config(config_template: str):
...
         machine_id = Conf.machine_id
         server_info = Utils._get_server_info(config_template_index, machine_id)
         if server_info is None:
-            Log.error(f"Could not find server information in {config_template}")
+            Log.error(f"Could not find server information in {config_path}")
             raise SetupError(errno.EINVAL, "Could not find server " +\
-                "information in %s", config_template)
+                "information in %s", config_path)
         config = CortxConf.get('message_bus')
         Utils._create_utils_config(server_info, machine_id, \
```
Comment (sachinpunadikar) [inline py-utils/src/setup/utils.py:239]: Shouldn't we remove this ?

Comment (selvakumaar5496) [reply py-utils/src/setup/utils.py:239]: Shall we remove creation of config file? I have just fixed the issue.

Comment (selvakumaar5496) [reply py-utils/src/setup/utils.py]: Done, removed

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by veerendra-simha-garikipati — APPROVED** [review]
+1

**Review by suryakumar1024 — APPROVED** [review]

**Pull request merged**